nucliadb/nucliadb/ingest/service/writer.py

```diff
@@ -741,6 +745,118 @@ async def generate_buffer(
         result = FileUploaded()
         return result
 
+    async def CreateShadowShard(  # type: ignore
```

Comment:
Should be created on the Node class, and not implemented on LocalNode

Reply:
done